src/components/style/modals/menu.css

```diff
@@ -0,0 +1,29 @@
+/**
+ * Overrides for @reach/menu-button
+ */
+
+[data-reach-menu-list] {
+  border-width: 0;
+  padding: 0;
+  font-size: inherit;
+  min-width: 140px;
+  max-width: 460px;
+  overflow-y: scroll;
+}
+
+[data-reach-menu-item] {
+  padding: var(--space-half) var(--space-default);
+  white-space: initial;
+  margin: 0;
+}
+
+/* use highlight color instead of focus ring */
+[data-reach-menu-item]:focus {
+  outline: 0; /* stylelint-disable-line */
+}
+
+[data-reach-menu-item][data-selected] {
+  background: var(--color-lightGray);
+  color: inherit;
+}
+
```

Comment:
One note here: all the components in Reach UI seem to spread props, so the className could be provided on the components. I don't have a strong preference here though.

Reply:
I'm following the styling guide they published, which recommends using these attribute selectors. It's a total gamble, but my bet is that this method of styling will most likely remain stable in future updates of the library. I could be wrong, though